Framework/src/AggregatorRunner.cxx

```diff
@@ -247,6 +260,14 @@ void AggregatorRunner::store(QualityObjectsType& qualityObjects)
   }
 }
 
+void AggregatorRunner::send(const core::QualityObjectsType& qualityObjects, framework::DataAllocator& allocator)
+{
+  ILOG(Debug, Devel) << "Sending" << qualityObjects.size() << " quality objects" << ENDM;
+  for (const auto& qualityObject : qualityObjects) {
+    allocator.snapshot(framework::Output{ createDataOrigin(), createAggregatorRunnerDataDescription(createAggregatorRunnerName()), 0 }, *qualityObject);
```

Comment:
I would propose to stay in line with how `QualityObjects` are published by `CheckRunner`. In particular, I mean that each `Aggregator` should have it's own output to send to. This will be consistent with `CheckRunner`, but most importantly, it will allow `BookkeepingQualitySink` to subscribe only to the inputs it needs.

Reply:
okay, will do